Tests/Async/CacheResolvedTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Liip\ImagineBundle\Tests\Async;
+
+use Liip\ImagineBundle\Async\CacheResolved;
+
+class CacheResolvedTest extends \PHPUnit_Framework_TestCase
+{
+    public static function setUpBeforeClass()
+    {
+        if (false == getenv('WITH_ENQUEUE')) {
```

Comment:
Again, I'd prefer this testing wasn't based on an environment variable but globally run if the required bundle exists.